src/main/java/edu/kis/powp/jobs2d/drivers/fabrics/CommandFactory.java

```diff
@@ -0,0 +1,19 @@
+package edu.kis.powp.jobs2d.drivers.fabrics;
+
+import edu.kis.powp.jobs2d.Job2dDriver;
+import edu.kis.powp.jobs2d.drivers.command.ComplexCommand;
+import edu.kis.powp.jobs2d.drivers.command.SetPositionCommand;
+
+public class CommandFactory {
+
+    public ComplexCommand makeTriangle(Job2dDriver driver) {
```

Comment:
DriverCommand return type would be sufficient.
This factory is too small and there is no test verifying how everything works. Missing point 3.1.5.